src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,96 @@
+package seedu.addressbook.ui;
+
+import static seedu.addressbook.common.Messages.MESSAGE_GOODBYE;
+import static seedu.addressbook.common.Messages.MESSAGE_INIT_FAILED;
+import static seedu.addressbook.common.Messages.MESSAGE_PROGRAM_LAUNCH_ARGS_USAGE;
+import static seedu.addressbook.common.Messages.MESSAGE_USING_STORAGE_FILE;
+import static seedu.addressbook.common.Messages.MESSAGE_WELCOME;
+
+import java.util.*;
+import seedu.addressbook.commands.*;
+
+public class Formatter {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.